--- origsrc/urllib3-1.7.1/setup.py	2013-06-26 13:39:49.000000000 -0500
+++ src/urllib3-1.7.1/setup.py	2014-02-05 19:22:18.068991200 -0600
@@ -44,7 +44,7 @@ setup(name='urllib3',
       author_email='andrey.petrov@shazow.net',
       url='http://urllib3.readthedocs.org/',
       license='MIT',
-      packages=['urllib3', 'dummyserver',
+      packages=['urllib3',
                 'urllib3.packages', 'urllib3.packages.ssl_match_hostname',
                 'urllib3.contrib',
                 ],
--- origsrc/urllib3-1.7.1/urllib3/connectionpool.py	2013-09-25 11:01:09.000000000 -0500
+++ src/urllib3-1.7.1/urllib3/connectionpool.py	2014-02-05 19:22:18.071991600 -0600
@@ -54,8 +54,11 @@ from .exceptions import (
     ReadTimeoutError,
     ProxyError,
 )
-from .packages.ssl_match_hostname import CertificateError, match_hostname
-from .packages import six
+try:
+    from ssl import CertificateError, match_hostname
+except ImportError:
+    from .packages.ssl_match_hostname import CertificateError, match_hostname
+import six
 from .request import RequestMethods
 from .response import HTTPResponse
 from .util import (
--- origsrc/urllib3-1.7.1/urllib3/fields.py	2013-08-14 16:43:05.000000000 -0500
+++ src/urllib3-1.7.1/urllib3/fields.py	2014-02-05 19:22:18.074992000 -0600
@@ -7,7 +7,7 @@
 import email.utils
 import mimetypes
 
-from .packages import six
+import six
 
 
 def guess_content_type(filename, default='application/octet-stream'):
--- origsrc/urllib3-1.7.1/urllib3/filepost.py	2013-08-14 16:43:05.000000000 -0500
+++ src/urllib3-1.7.1/urllib3/filepost.py	2014-02-05 19:22:18.077492300 -0600
@@ -10,8 +10,8 @@ import mimetypes
 from uuid import uuid4
 from io import BytesIO
 
-from .packages import six
-from .packages.six import b
+import six
+from six import b
 from .fields import RequestField
 
 writer = codecs.lookup('utf-8')[3]
--- origsrc/urllib3-1.7.1/urllib3/response.py	2013-09-25 11:01:09.000000000 -0500
+++ src/urllib3-1.7.1/urllib3/response.py	2014-02-05 19:22:18.080492700 -0600
@@ -10,7 +10,7 @@ import zlib
 import io
 
 from .exceptions import DecodeError
-from .packages.six import string_types as basestring, binary_type
+from six import string_types as basestring, binary_type
 from .util import is_fp_closed
 
 
--- origsrc/urllib3-1.7.1/urllib3/util.py	2013-09-25 11:01:09.000000000 -0500
+++ src/urllib3-1.7.1/urllib3/util.py	2014-02-05 19:22:18.083993100 -0600
@@ -32,7 +32,7 @@ try:  # Test for SSL features
 except ImportError:
     pass
 
-from .packages import six
+import six
 from .exceptions import LocationParseError, SSLError, TimeoutStateError
 
 
